### PR TITLE
feat: Galician Locale

### PR DESCRIPTION
**Pull request by barreeeiroo**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Other (about what?): locale update

### 🔗 Related issue link

Adds support for Galician language.

### 💡 Background and solution

This PR intends to add support for another language, Galician, spoken in the northwest of Spain.

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
There is no `rc-picker/lib/locale/gl_ES` now.

**Comment by barreeeiroo** [comment]
@afc163 I know, I've also submitted a PR to their repo, so it can be used here

**Comment by yoyo837** [comment]
> @afc163 I know, I've also submitted a PR to their repo, so it can be used here

Is it submitted to the wrong place？

https://github.com/react-component/picker

**Comment by barreeeiroo** [comment]
@yoyo837 My bad, I did only the rc-pagination, not the rc-picker. Opened a PR now
https://github.com/react-component/picker/pull/120

**Comment by afc163** [comment]
https://github.com/react-component/picker/releases/tag/v2.0.4

**Comment by yoyo837** [comment]
Now you need to switch to the `feature` branch and dependence on the new version of `rc-picker`.

**Comment by afc163** [comment]
ping @barreeeiroo

**Comment by barreeeiroo** [comment]
@yoyo837 @afc163 fixed, and rebased the branch onto `feature`. I bumped the `rc-picker` to the minimum required version to work with the Galician locale.
All checks are passing.

**Comment by yoyo837** [comment]
`docs/react/i18n.en-US.md`
`docs/react/i18n.zh-CN.md`

Please add docs for it.

**Comment by afc163** [comment]
Please add test case.

File: docs/react/i18n.zh-CN.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ return (
 | 法语（比利时）       | fr_BE  |
 | 法语（法国）         | fr_FR  |
 | 爱尔兰语             | ga_IE  |
+| 加利西亚人           | gl_ES  |
```
Comment (yoyo837) [inline docs/react/i18n.zh-CN.md:47]:
```suggestion
| 加利西亚语（西班牙）           | gl_ES  |
```

**Comment by yoyo837** [comment]
Pls run:

`npm test -- -u`

and push all changes.

**Comment by afc163** [comment]
Please add test case like other locales.

**Comment by barreeeiroo** [comment]
@afc163 which test case are you talking about? I searched for `es_ES` tests, and only found one.

**Comment by afc163** [comment]
1. Please rebase **latest** feature branch and run `npm test -- -u` to update test snapshot.
2. Please add a semantic commit or PR title.

image

**Comment by barreeeiroo** [comment]
@afc163 it should be ready now. I have changed the PR name and made a commit with a semantic name (which now passes the check), and run `npm test -- -u` updating the test snapshots. It now uses the latest `feature` commit:

image

**Comment by zombieJ** [comment]
@barreeeiroo, feature branch merged master yet. Pls rebase again. Thanks : )

**Comment by yoyo837** [comment]
1. rebase, again :)
2. `git clean -fdx` or delete `node_modules`, `package-lock.json` or `yarn.lock` manually
3. `npm test -- -u`
4. push all changes

**Review by zombieJ — APPROVED** [review]

**Pull request merged**